content/operate/rc/changelog/april-2024.md

```diff
@@ -12,6 +12,12 @@ linktitle: April 2024
 weight: 55
 ---
 
+## New features
+
+### Nano billing unit
+
+We added a Nano billing unit for Pro databases with a max. size of 500GB and max. throughput of 5K ops/sec. Use it to create smaller databases for a lower cost.
```

Comment:
Presumably abbreviations like "max" are OK in the context of a changelog? It would look a bit odd in a normal doc page.

Reply:
You're right. I thought it might match our updated voice and tone to shorten it to max., but I agree that it reads strangely.

```suggestion
We added a Nano billing unit for Pro databases with a maximum size of 500GB and a maximum throughput of 5K ops/sec. Use it to create smaller databases for a lower cost.
```